scripts/test/SceneBuilder.ps1

```diff
@@ -0,0 +1,300 @@
+   $UnityPath = ${Env:${{ parameters.UnityVersion }}}
+   # Find unity.exe as Start-UnityEditor currently doesn't support arbitrary parameters
+   $editor = Get-ChildItem $UnityPath -Filter 'Unity.exe' -Recurse | Select-Object -First 1 -ExpandProperty FullName
+   
+   # The build output goes to a unique combination of Platform + Arch + Scene to ensure that
+   # each build will have a fresh destination folder.
+   $outDir = "$(Build.ArtifactStagingDirectory)\build\${{ parameters.UnityVersion }}\${{ parameters.Platform }}_${{ parameters.Arch }}_${{ parameters.Scene }}"
+   $logFile = New-Item -Path "$outDir\build\build.log" -ItemType File -Force
+   $logDirectory = "$outDir\logs"
+   $projectPath = $(Get-Location)
+   $sceneList = ''
+   if ("${{ parameters.Scene }}" -eq "HandInteractionExamples")
+   {
+       $sceneList += 'Assets\MRTK\Examples\Demos\HandTracking\Scenes\HandInteractionExamples.unity'
+   }
+   elseif ("${{ parameters.Scene }}" -eq "SolverExamples")
```

Comment:
It's interesting to see that these are all elseifs, but sceneList += is done. This doesn't seem to be doing the thing of adding multiple right?

Reply:
That would be me using the $extraArgs chunk and using that as my base for the layout of the $sceneList elseifs and overlooking the + in front of the = when I reviewed this. This is not supposed to be adding multiple. At least not yet, some changes need to be made to address the large number of steps in pipelines/templates/buildcallsetup.yml and that may include moving the steps into PowerShell from pipelines.